packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -792,6 +855,26 @@ class AccountInternal extends PureComponent {
       }
     };
 
+    if (name === 'amount' || name === 'payee') {
+      let { data } = await runQuery(
+        q('transactions')
+          .filter({ id: { $oneof: ids }, reconciled: true })
+          .select('*')
+          .options({ splits: 'grouped' }),
+      );
+      let transactions = ungroupTransactions(data);
+
+      if (transactions.length > 0) {
+        this.props.pushModal('confirm-transaction-edit', {
+          onConfirm: () => {
+            this.props.pushModal('edit-field', { name, onSubmit: onChange });
+          },
+          confirmReason: 'batchEditWithReconciled',
+        });
+        return;
+      }
+    }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I'm not quite sure I understand this. Could you please explain?

Reply:
If you are performing a batch edit it scans to check if any of the selected transactions are locked. If they are, and you are editing a protected field (amount or payee) then it pops the warning first.

Reply:
Makes sense. Thanks!